src/index.js

```diff
@@ -8,11 +8,9 @@ import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-    <React.StrictMode>
```

Comment:
The problem with using React.StrictMode in our case is that it calls all lifecycle function twice even if they should be called only once. refer to this :- https://stackoverflow.com/questions/60618844/react-hooks-useeffect-is-called-twice-even-if-an-empty-array-is-used-as-an-ar
Since we initialize map in useEffect it causes a problem of reinitialization. This wont be a problem in production environment but it is a problem in dev.